### ISSUE TITLE
LibreOffice apps do not open (znver1 systems only)

### ISSUE THREAD

**Issue by benbullard79** [OpenMandrivaAssociation/distribution#2920]
**OpenMandriva version**:
Cooker and ROME (This report is from ROME Plasma z**nver1 system** installed 2023-06-02 and fully dsynced)
**Describe the bug**:
LibreOffice apps fail to start.
**Steps to reproduce**:
In **znver1 cooker or rome system** from Konsole:
`$ lowriter`
or: `libreoffice`, `loimpress`, `lomath`, `localc`, or `lodraw`.
**Observed behavior**: 
Example output:
```
$ localc
javaldx: Could not find a Java Runtime Environment!
Warning: failed to read path from javaldx
Failed to create wl_display (No such file or directory)
qt.qpa.plugin: Could not load the Qt platform plugin "wayland" in "" even though it was found.
Unspecified Application Error
```
**Expected behavior**:
That each LibreOffice application open and is functional.

**Additional comment**:

**Logs and screenshots if relevant**
Journalctl output after attempting to start the various LO applications.

[journal-lo_not_starting.txt](https://github.com/OpenMandrivaAssociation/distribution/files/11643505/journal-lo_not_starting.txt)

```
$ rpm -qa | grep libreoffice
libreoffice-help-en_US-7.5.3.2-2.znver1
libreoffice-style-breeze-7.5.3.2-2.znver1
libreoffice-common-7.5.3.2-2.znver1
libreoffice-draw-7.5.3.2-2.znver1
libreoffice-help-es-7.5.3.2-2.znver1
libreoffice-l10n-es-7.5.3.2-2.znver1
libreoffice-style-breeze_dark-7.5.3.2-2.znver1
libreoffice-kde5-7.5.3.2-2.znver1
libreoffice-impress-7.5.3.2-2.znver1
libreoffice-calc-7.5.3.2-2.znver1
libreoffice-math-7.5.3.2-2.znver1
libreoffice-writer-7.5.3.2-2.znver1
```

[omdv-bug-report-20230603140759.log](https://github.com/OpenMandrivaAssociation/distribution/files/11643519/omdv-bug-report-20230603140759.log)

**Comment by benbullard79** [comment]
In Cooker this issue seems to have occurred between April 1 and April 22. Between Cooker znver1 iso # 1867 and # 1888.

**Comment by rugyada** [comment]
Fixed?

**Comment by benbullard79** [comment]
Fixed w/version libreoffice-7.6.1.2-1.
